packages/block-library/src/navigation/index.php

```diff
@@ -134,6 +134,8 @@ private static function get_markup_for_inner_block( $inner_block ) {
 		if ( ! empty( $inner_block_content ) ) {
 			if ( static::does_block_need_a_list_item_wrapper( $inner_block ) ) {
 				return '<li class="wp-block-navigation-item">' . $inner_block_content . '</li>';
+			} else {
+				$inner_block_content = '';
 			}
```

Comment:
I'm not following this change.

The `$inner_block_content` "not empty". So we then check whether it needs a wrapper `<li>`. If it doesn't we return an empty string. Does that make sense?

To me we need to ensure that the method returns a string as per the `@return`. 

Therefore if `$inner_block_content` is empty we should return early with an empty string, and only allow the remainder of the method to execute if there is content.

Reply:
Yea, I only implemented the text in the issue. Will revise